feature_tests/c/include/AttrEnum.h

```diff
@@ -20,4 +20,5 @@
 
 
 
+
```

Comment:
nit: would be nice if we avoided this extra blank line generation in the c backend